test/Npgsql.CrateDb.Tests/CrateDbIntegrationTest.cs

```diff
@@ -0,0 +1,191 @@
+
+                    cmd.Parameters.AddWithValue("@id", 1);
+                    cmd.Parameters.AddWithValue("@str_array", new string[] { "a", "b", "c", "d" });
+                    cmd.Parameters.AddWithValue("@bool_array", new bool[] { true, false });
+                    cmd.Parameters.AddWithValue("@byte_array", new byte[] { 120, 100 });
+                    cmd.Parameters.AddWithValue("@short_array", new short[] { 1300, 1200 });
+                    cmd.Parameters.AddWithValue("@integer_array", new int[] { 2147483647, 234583 });
+                    cmd.Parameters.AddWithValue("@long_array", new long[] { 9223372036854775807L, 4L });
+                    cmd.Parameters.AddWithValue("@float_array", new float[] { 3.402f, 3.403f, 1.4f });
+                    cmd.Parameters.AddWithValue("@double_array", new double[] { 1.79769313486231570e+308, 1.69769313486231570e+308 });
+                    cmd.Parameters.AddWithValue("@timestamp_array", new DateTime[] { new DateTime(2000, 1, 1).ToUniversalTime(), new DateTime(1970, 1, 1).ToUniversalTime() });
+                    cmd.Parameters.AddWithValue("@ip_array", new string[] { "127.142.132.9", "127.0.0.1" });
+                    cmd.Parameters.AddWithValue("@obj_array", NpgsqlTypes.NpgsqlDbType.Json, "[ { \"inner\": \"Zoon1\" }, { \"inner\": \"Zoon2\" } ]");
+                    //cmd.Parameters.AddWithValue("@obj_array", NpgsqlTypes.NpgsqlDbType.Json, new object[] { new { inner = "Zoon1" }, new { inner = "Zoon2" } });
+                    cmd.Parameters.AddWithValue("@geo_shape_array", new string[] { "POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))", "POLYGON ((40 20, 50 50, 30 50, 20 30, 40 20))" });
+                    cmd.ExecuteNonQuery();
+
+                    cmd.CommandText = "refresh table arrayTest";
+                    cmd.ExecuteNonQuery();
+                }
+            }
+        }
+
+        protected void TearDownArrayTestTable()
+        {
+            using (var con = OpenConnection())
+            {
+                using (var cmd = con.CreateCommand())
+                {
+                    cmd.CommandText = @"drop table if exists arrayTest";
+                    cmd.ExecuteNonQuery();
+                }
+            }
+        }
+
+        protected void CreateBulkInsertTestTable()
+        {
+            using (var con = OpenConnection())
+            {
+                using (var cmd = con.CreateCommand())
+                {
+                    cmd.CommandText = @"create table if not exists authors(
+                        id integer primary key,
+                        author string                        
+                      ) clustered by (id) into 1 shards with (number_of_replicas=0)";
+
+                    cmd.ExecuteNonQuery();
+                }
+            }
+        }
+
+        protected void TearDownBulkInsertTestTable()
+        {
+            using (var con = OpenConnection())
+            {
```

Comment:
Other tests omit these for consecutive `using` statements. Either way is fine, but would align for consistency.

Reply:
i reformatted it. Thx